### DISCUSSION TITLE
Similar Program for OnlyFans & how to send Donations?

### DISCUSSION THREAD

**Discussion by PrettyLights808** [Avnsx/fansly#129] [Q&A]
This downloader is by far the best one I've come across. There's ones similar for OnlyFans, but it's not nearly as easy to setup and use. I would love to see this same program but for OnlyFans. What would it take to do that?

**Comment by Avnsx** [comment]
>  I would love to see this same program but for OnlyFans. What would it take to do that?

Projects like these require alot of maintenance, due to continously having to be up to date with the scraped websites backend.

Back in the day I made this project, purely because I wanted to have a popular project with lots of users, as the thought of thousands of people using my code on a daily was fascinating to me.

Luckily with fansly, the maintenance required is moderate. On the other hand OnlyFans, is a way bigger company with way more resources, thus they have the capacity to have people who are entirely on the payroll to make projects like these drastically harder to maintain, by continously and way more frequently pushing security patches with the intention to break the functionality offered through tools, similar to mine. Resulting in the open source maintainers needing to counteract on those, so their programs continue to work and it just ends up being a infinitely repeating cat and mouse game, where neither side is really gaining anything, except one party is getting paid for the time consumed and the other is basically just wasting their time for minimum RoI.

From my experience, the audience for such projects is unfortunately not interested in supporting the work of the maintainers either. They are primarily interested in the functionality provided and do not show appreciation for the work that goes into creating and maintaining it.

For example, despite having over 30,000 users, I have only received a single donation. This can be attributed in part to the efforts of large corporations that collaborate with companies like OnlyFans or Fansly to intimidate payment processors and prevent them from providing service to repository owners of such scraping projects. This includes public donations through widgets such as "Buy me a Coffee" buttons on the repository itself. However, it is also due to the fact that people simply do not value the work of the maintainers enough to support them.

Further on, in the current era, censorship is prevalent. For instance, this project has been blacklisted from the Google search engine for relevant high SEO strength search terms such as "Fansly Downloader." As a result, it receives far less recognition than it deserves due to not being indexed for these terms.

Given these numerous obstacles, it is highly unlikely that I will create a similar project for OnlyFans. However, if my experience with this current project improves and people begin to recognize its value and show more support for the specific niche use case it fulfills, I may reconsider.

**Comment by LeroyDaydreamer** [comment]
This explains a lot! Though it's not obvious for me, how one even could donate. I couldn't find any clue or even encouragement for this. Either way, thank you, your program works great!

**Reply by Avnsx** [reply]
As previously stated, I am unable to add any donation methods to my repository or github profile due to restrictions from payment processors, who have been contacted by representatives of large corporations and been told to cease their service. 

However, if you would like to generally support me for "my continuous efforts on GitHub", you can reach out to me privately via email at ``AvnDev@protonmail.com``, then I could provide you with easy & common ways to donate.

Thank you for your support, in-case you decide to do so! 👍
